devproject/celery.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Misago requires Python 3.6 to work - those imports are unnecessary and can be removed.